chrysom/fx.go

```diff
@@ -4,89 +4,11 @@
 package chrysom
 
 import (
-	"context"
-	"errors"
-
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/xmidt-org/touchstone"
 	"go.uber.org/fx"
-	"go.uber.org/zap"
 )
 
-// GetLogger returns a logger from the given context.
-type GetLogger func(context.Context) *zap.Logger
-
-// SetLogger embeds the `Listener.logger` in outgoing request contexts for `Listener.Update` calls.
-type SetLogger func(context.Context, *zap.Logger) context.Context
-
-type BasicClientIn struct {
-	fx.In
-
-	// Ancla Client config.
-	Config BasicClientConfig
-	// GetLogger returns a logger from the given context.
-	GetLogger GetLogger
```

Comment:
Most code was moved to `ancla/chrysom/listener/fx.go` while the `ProvideBasicClient` related code was moved to `ancla/fx.go`